QMK support pcb db63 hotswap rbg (#7771)
qmk/qmk_firmware: keyboards/db/db63/README.md

# DB63

A 65% keyboard hotswap with RGB that runs ps2avrgb natively, with USB C, RGB underglow and backlight.
Firmware is still in **beta**

* Keyboard Maintainer: QMK Community

> * Hardware Supported: 

Reviewer: ```suggestion
* Hardware Supported: DB63 (ATmega32A)
```

Author: updated in the new commit

> * Hardware Availability: 

Reviewer: If the PCB is for sale anywhere, a link to the product page should go here. Something like:

```suggestion
* Hardware Availability: [DB Store](https://dbstore.vn/)
```

Otherwise, delete this line.

Author: The board right now is out of stock, maybe i will remove this, thanks

Make example for this keyboard (after setting up your build environment):

    make db/db63:default

Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))

    make db/db63:default:flash

> **Reset Key**: Hold down the key located at *K00*, commonly programmed as *Escape* while plugging in the keyboard(*all backlight led will flashing which indicate the board is in bootloader mode*)

Reviewer: Tiny typo and a bit of English "grammaring." :slightly_smiling_face: 

```suggestion
**Reset Key**: Hold down the key located at *K00*, commonly programmed as *Escape* while plugging in the keyboard. (*All backlight LEDs will flash which indicate the board is in bootloader mode.*)
```

> **Reset Key**: Hold down the key located at *K00*, commonly programmed as *left control* while plugging in the keyboard. Another trick to enable the board in reset mode is using BootMapper Client->Options->Set Bootloader

Reviewer 2: > Another trick to enable the board in reset mode is using BootMapper Client->Options->Set Bootloader

Once flashed with QMK, this wont work any more due to #7456.

> **Reset Key**: Hold down the key located at *K00*, commonly programmed as *left control* while plugging in the keyboard. Another trick to enable the board in reset mode is using BootMapper Client->Options->Set Bootloader (this is only available for the first times before flashing QMK firmware)

Reviewer: Is the stock Reset key locked to B5 × A0? Because that would be `K40` and not `K00`. Otherwise, the `*left control*` here should be `*Escape*`. which *is* `K00`.

This paragraph should almost definitely be expanded if the flashing procedure changes after QMK is installed on the board.

Author: It's the *Escape* key, i will update this paragraph

> **Tips**: Another trick to enable the board in reset mode is using BootMapper Client->Options->Set Bootloader (*this is only available for the first times before flashing QMK firmware*)

Reviewer: ```suggestion
**Tips**: Another trick to enable the board in reset mode is using BootMapper Client->Options->Set Bootloader. (*This is only available for the first time, before flashing QMK Firmware.*)
```
